WcaOnRails/config/routes.rb

```diff
@@ -58,7 +58,13 @@
   get 'polls/:id/vote' => 'votes#vote', as: 'polls_vote'
   get 'polls/:id/results' => 'polls#results', as: 'polls_results'
 
-  resources :teams, only: [:index, :new, :create, :update, :edit]
+  resources :committees do
+    resources :teams, only: [:new, :edit, :create, :update, :destroy] do
+      resources :team_members, only: [:new, :edit, :create, :update]
+    end
+    resources :committee_positions, only: [:index, :new, :edit, :create, :update, :destroy]
```

Comment:
Ah I see. Actually I've stumbled upon identical issue in my most recent PR. The way I managed to fix it was to pass variale `url` to the form partial and then use it with simple form `url: url`.